packages/element-web-playwright-common/playwright-screenshots.sh

```diff
@@ -69,15 +85,47 @@ done
 build_image
 
 # Ensure we pass all symlinked node_modules to the container
-pushd node_modules || exit > /dev/null
+pushd node_modules > /dev/null
 SYMLINKS=$(find . -maxdepth 2 -type l -not -path "./.bin/*")
-popd || exit > /dev/null
+popd > /dev/null
 for LINK in $SYMLINKS; do
-  TARGET=$(readlink -f "node_modules/$LINK")
+  TARGET=$(readlink -f "node_modules/$LINK") || true
   if [ -d "$TARGET" ]; then
+     if docker --version | grep -q podman; then
+        echo -e "\033[31m" >&2
+        cat <<'EOF' >&2
+WARNING: `node_modules` contains symlinks, and the support for this in
+`playwright-screenshots.sh` is broken under podman due to
+https://github.com/containers/podman/issues/25947.
+
+If you get errors such as 'Error: crun: creating `<path>`', then retry this
+having `yarn unlink`ed the relevant node modules.
+EOF
+      echo -e "\033[0m" >&2
+    fi
     echo "mounting linked package ${LINK:2} in container"
     RUN_ARGS+=( "-v" "$TARGET:/work/node_modules/${LINK:2}" )
   fi
 done
 
+# Our Playwright fixtures use Testcontainers [1], which uses a docker image
+# called Ryuk [2], which will clean up any dangling containers/networks/etc
+# after a timeout, if the parent process dies unexpectedly.
+#
+# To do this, Ryuk requires access to the docker socket, so Testcontainers
+# starts the Ryuk container with a bind-mount of `/var/run/docker.sock`.
```

Comment:
it seems to query the socket location from the Docker API rather than hard-coding `/var/run/docker.sock`, no?